cursorless.nvim/lua/cursorless/init.lua

```diff
@@ -0,0 +1,131 @@
+-- local Config = require('cursorless.config')
+
+-- We can't use that yet as then when we use load_extensions() we get an error.
```

Comment:
Let's put this code into a github issue if it is important to keep it for some reason. We don't like to have code commented out in the repo

Reply:
done